### DISCUSSION TITLE
Нет подключения к Интернету

### DISCUSSION THREAD

**Discussion by Ponboyy** [bol-van/zapret#868] [Q&A]
Видео на ютубе грузятся, но сам сайт выдает ошибку и пишет Нет подключения к Интернету

**Comment by Fenrir-0xFF** [comment]
В списке все эти сайты есть? 
_youtube.com
googlevideo.com
ytimg.com
ggpht.com
youtubei.googleapis.com_

И надеюсь не яндекс браузер у тебя?

**Comment by Ponboyy** [comment]
> В списке все эти сайты есть? 
> _youtube.com
> googlevideo.com
> ytimg.com
> ggpht.com
> youtubei.googleapis.com_
> 
> И надеюсь не яндекс браузер у тебя? 



Chrome у меня, да эти сайты есть

**Comment by TeslaWP** [comment]
та же херня

**Comment by numbereleven-a** [comment]
Прогнать блокчек для ютуба, по возможности найти стратегию рабочую как для youtube.com так и для ggc хостов с видео, при невозможности разделить стратегии.
